index.bs

```diff
@@ -1100,6 +1100,16 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
             1. Return |constructCredentialAlg| and terminate this algorithm.
 
+        :   If the [=client=] concludes that it is impossible for any eligible [=authenticator=] to appear,
+        ::  The client MAY prompt the user for [=user consent|consent=] to [=authentication|authenticate=] to the [=[RP]=]. If
```

Comment:
I believe that the first "The" should be "the".

Reply:
At first I thought so too - but all the other cases in this switch statement use this style, so I changed this one to do that too for consistency. Let me know if you want me to revert it to lowercase.

Reply:
offhand, IMHO, we should use ":" at the end of the switch stmt title rather than "," -- fwiw, using comma may be an artifact of my having just blindly following the style established by other more w3c-experienced editors and not thinking about it...